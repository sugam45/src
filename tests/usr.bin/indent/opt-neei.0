/* $NetBSD: opt-neei.0,v 1.2 2021/03/06 23:09:17 rillig Exp $ */
/* $FreeBSD$ */

bool
less(int a, int b)
{
	if (a <
	    b)
		return true;
	if (a
	    <
	    b)
		return true;
}
