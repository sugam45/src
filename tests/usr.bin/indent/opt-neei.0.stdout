/* $NetBSD: opt-neei.0.stdout,v 1.2 2021/03/06 23:09:17 rillig Exp $ */
/* $FreeBSD$ */

bool
less(int a, int b)
{
	if (a <
	    b)
		return true;
	if (a
	    <
	    b)
		return true;
}
